README.md

```diff
@@ -26,6 +26,10 @@ it's possible to run them in your local environment. You just need to make sure
 between your environment and a Kubernetes cluster works well. This is a default config file of these 
 components, please see also [the sample file](config/samples/kubesphere.yaml).
 
+### Get Jenkins password
+```shell
+kubectl get secret --namespace kubesphere-devops-system ks-ctl-jenkins -o jsonpath="{.data.jenkins-admin-password}" | base64 --decode;echo
+```
```

Comment:
I prefer to keep the statement in the Helm chart. See also https://github.com/kubesphere-sigs/ks-devops/blob/master/charts/ks-devops/templates/NOTES.txt#L5

Because the command might be different. For example, users can provide a different namespace.

Reply:
fixed